internal/services/order/stocks_feed.go

```diff
@@ -0,0 +1,24 @@
+package order
```

Comment:
why this method is in order service?

Reply:
then can we keep it as a separate file itself in the services folder?

Reply:
oh.. you have named transaction engine as order service. Is that so?